wallets/react-wallet-v2/src/utils/HelperUtil.ts

```diff
@@ -121,10 +121,17 @@ export function isElrondChain(chain: string) {
 /**
  * Check if chain is part of TRON standard
  */
- export function isTronChain(chain: string) {
+export function isTronChain(chain: string) {
   return chain.includes('tron')
 }
 
+/**
+ * Check if chain is part of Tezos standard
+ */
+export function isTezosChain(chain: string) {
+  return chain.includes('tezos')
```

Comment:
is it better to use `startsWith` instead of `includes`?

Reply:
I copied the usage from the methods of the other chains, I guess if we change it here we should change it for all of them.